components/pages/ProfileTab.js

```diff
@@ -32,9 +36,15 @@ const styles = StyleSheet.create({
   },
 });
 
-const sampleData = [{
-  username: 'Swolebraham Lincoln',
-  fitnessStats: [
+const sampleData = {
+  email: 'test@ai.com',
+  name: {
+    first: 'Test',
+    last: 'Ai',
+  },
+  timeStamp: new Date(),
+  username: 'Test AI',
+  stats: [
```

Comment:
okay for main so that others can navigate without errors, but for your own branch let's clear out all uses of sample data so we're forced to find and fix areas where our logic has holes. e.g. I'd rather find that we have an error when trying to render an empty list of user info than cover it by rendering fake data.